scripts/on-release.js

```diff
@@ -10,8 +10,10 @@ const hash = require('crypto').createHash('sha256');
 // RAM
 const sha256 = hash.update(fs.readFileSync(artifact)).digest('hex');
 
-shell.sed(
-    '-i', 'download/[0-9\.]*/rules_nodejs-[0-9\.]*.tar.gz',
-    `download/${version}/rules_nodejs-${version}.tar.gz`, 'README.md');
-shell.sed('-i', 'sha256 = \"[0-9a-f]+\"', `sha256 = "${sha256}"`, 'README.md');
+for (const f in ['README.md', 'packages/create/index.js']) {
+  shell.sed(
+      '-i', 'download/[0-9\.]*/rules_nodejs-[0-9\.]*.tar.gz',
```

Comment:
thanks for commenting! we don't get enough thoughtful review on a lot of these PRs, so it's a great way to help 😎 

the 0.0.0-PLACEHOLDER can't be in the README because then it would appear that way on the github site (where you view sources, not the published distribution)

also for the other replacement, we don't know the sha256 of the package until after we've built it, so it's too late for bazel templating to help there